modules/ROOT/pages/amazon/amazon-sqs-connector-reference.adoc

```diff
@@ -772,21 +776,92 @@ Sets the value of one or more queue attributes. When you change a queue's attrib
+| Revocation Check a| * <<standard-revocation-check>>
+* <<custom-ocsp-responder>>
+* <<crl-file>> | Validates that a certificate was revoked. |  |
+|======================
+
+[[TrustStore]]
+=== Trust Store
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location of the trust store. The path is resolved relative to the current classpath and file system, if possible. |  |
+| Password a| String | The password used to protect the trust store. |  |
+| Type a| String | The type of store used. |  |
+| Algorithm a| String | The algorithm used by the trust store. |  |
+| Insecure a| Boolean | If `true`, no certificate validations are performed, which makes connections vulnerable to attacks. Use at your own risk. |  |
+|======================
+
+[[KeyStore]]
+=== Key Store
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location of the key store. The path is resolved relative to the current classpath and file system, if possible. |  |
+| Type a| String | The type of store used. |  |
+| Alias a| String | The alias of the key to use when the key store contains multiple private keys. If not defined, the first key in the file is used by default. |  |
+| Key Password a| String | The password used to protect the private key. |  |
+| Password a| String | The password used to protect the key store. |  |
+| Algorithm a| String | The algorithm used by the key store. |  |
+|======================
+
+[[standard-revocation-check]]
+=== Standard Revocation Check
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean | Only verify the last element of the certificate chain. |  |
+| Prefer Crls a| Boolean | Try CRL instead of OCSP first. |  |
+| No Fallback a| Boolean | Do not use the secondary checking method, which is the method not specified in the Prefer Crls field. |  |
+| Soft Fail a| Boolean | Avoids verification failure when the revocation server cannot be reached or is busy. |  |
+|======================
+
+[[custom-ocsp-responder]]
+=== Custom OCSP Responder
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Url a| String | The URL of the OCSP responder. |  |
+| Cert Alias a| String | The alias of the signing certificate for the OCSP response, if present. The alias must be in the trust store. |  |
+|======================
+
+[[crl-file]]
+=== CRL File
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The path to the CRL file. |  |
+|======================
 [[Reconnection]]
```

Comment:
Replace the reconnect block per our doc

Reply:
Hmm - I did that for Reconnect. You can see that below.